### ISSUE TITLE
Subversion_SVN_EXECUTABLE not found

### ISSUE THREAD

**Issue by fedorov** [QIICR/dcmqi#126]
when configuring with Slicer support enabled, I get this message running make:

```
-- Could NOT find Subversion (missing:  Subversion_SVN_EXECUTABLE)
CMake Error at /Users/fedorov/github/Slicer/Extensions/CMake/SlicerBlockUploadExtension.cmake:38 (message):
  Variable Subversion_SVN_EXECUTABLE is set to an inexistent directory or
  file ! []
Call Stack (most recent call first):
  /Users/fedorov/github/Slicer/Extensions/CMake/SlicerEnableExtensionTesting.cmake:23 (include)
  /Users/fedorov/local/builds/Slicer4-Debug/Slicer-build/UseSlicer.cmake:421 (include)
  CMakeLists.txt:221 (include)


-- Configuring incomplete, errors occurred!
```

Looks like `Subversion_SVN_EXECUTABLE ` is not inherited from the Slicer build properly.

**Comment by fedorov** [comment]
Clarification: this is happening when building dcmqi outside the Slicer build process.

**Comment by jcfr** [comment]
Thanks for the report.

Commit https://github.com/QIICR/dcmqi/commit/b3ea5fa05f204d2575e2c898b614159a9a56bda7  intended to fix this and workaround a know CMake issue, I will have an other look

**Comment by jcfr** [comment]
> building dcmqi outside the Slicer build process.

Do you mean that you are building dcmqi as a Slicer extension ? 

Using something like this to configure:

```
Slicer_DIR=/path/to/Slicer-Superbuild/Slicer-build
git clone git://github.com/QIICR/dcmqi/
mkdir dcmqi-Release && cd $_
cmake -DCMAKE_BUILD_TYPE:STRING=Release \
  -DSlicer_DIR:PATH=$Slicer_DIR \
  $(for opt in $(cat $Slicer_DIR/CMakeCache.txt | grep CMAKE_OSX | tr "\n" " "); do echo -n "-D$opt "; done) \
  ../dcmqi
```

**Comment by jcfr** [comment]
I was able to reproduce the issue.

**Comment by fedorov** [comment]
resolved by #127
